From 5d9ccf47cac2f9bd8d1f92695cf70a3c4c4ffa61 Mon Sep 17 00:00:00 2001
From: Chen Liang <cl@rock-chips.com>
Date: Fri, 2 Sep 2016 15:54:30 +0800
Subject: [PATCH] ARM64: rockchip: cpufreq-dt: enable
 CPUFREQ_HAVE_GOVERNOR_PER_POLICY

The cpu class need independent governor tunables for big/little system.

Change-Id: I0c31f694b6790c05c18b0c3d5522fa75143f293e
Signed-off-by: Chen Liang <cl@rock-chips.com>
---
 drivers/cpufreq/cpufreq-dt.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/cpufreq/cpufreq-dt.c b/drivers/cpufreq/cpufreq-dt.c
index dc6135e48b53..2c566060913e 100644
--- a/drivers/cpufreq/cpufreq-dt.c
+++ b/drivers/cpufreq/cpufreq-dt.c
@@ -349,7 +349,8 @@ static void cpufreq_ready(struct cpufreq_policy *policy)
 }
 
 static struct cpufreq_driver dt_cpufreq_driver = {
-	.flags = CPUFREQ_STICKY | CPUFREQ_NEED_INITIAL_FREQ_CHECK,
+	.flags = CPUFREQ_STICKY | CPUFREQ_NEED_INITIAL_FREQ_CHECK |
+		 CPUFREQ_HAVE_GOVERNOR_PER_POLICY,
 	.verify = cpufreq_generic_frequency_table_verify,
 	.target_index = set_target,
 	.get = cpufreq_generic_get,
-- 
2.35.3

